images/scripts/lib/build-deps.sh

```diff
@@ -14,12 +14,13 @@ OS_VER="$1"
 # most images use cockpit.spec from main branch, but there's a RHEL 7 stable branch with a completely different layout
 case "$OS_VER" in
     rhel*7|centos*7)
-        spec=$($GET "$COCKPIT_GIT/rhel-7.9/tools/cockpit.spec" |
-            sed 's/%{npm-version:.*}/0/; /Recommends:/d; s/build_optional 0/build_optional 1/')
+        url=$GET "$COCKPIT_GIT/rhel-7.9/tools/cockpit.spec"
```

Comment:
This can't work, it's a syntax error. Possibly you want to move the `$GET` into the next line?

```
❱❱❱ images/scripts/lib/build-deps.sh 'centos 7'
images/scripts/lib/build-deps.sh: line 17: https://raw.githubusercontent.com/cockpit-project/cockpit/rhel-7.9/tools/cockpit.spec: No such file or directory
```